index.html

```diff
@@ -2,455 +2,1416 @@
+              <h5 class="section-popular-section">Most popular</h5>
+              <h3 class="section-popular-title">
+                <a href="#">Amplitude celebrates 10 years with a free weekend of Endless games</a>
+              </h3>
+              <ul class="section-popular-byline mt-10">
+                <li><a href="#">Tristan Greene</a></li>
+                <li><span>4 days ago</span></li>
+              </ul>
+            </header>
+          </article>
+        </li>
+        <li>
+          <article class="section-latest">
+            <header class="section-latest-header">
+              <h5 class="orange-color section-latest-section">Latest</h5>
+              <h4>
+                <a class="text-decoration-none orange-color section-latest-title" href="#"> 
+                  The best vampire games and mods to play until Bloodlines 2 releases later this year                                      
+                </a>
+              </h4>
+            </header>
+            <a class="section-latest-image section-latest-image-9" href="#"></a>
+          </article>
+        </li>  
+        <li class="section-headline">
+          <h4 class="section-headline-title">
+            <a href="#">         
+              Games to play on date night: Rule the galaxy together in Stellaris                                    
+            </a>
+          </h4>
+        </li>
+        <li class="section-headline">
+          <h4 class="section-headline-title">
+            <a href="#">         
+              We need to make remote meetings more boring in 2021                                      
+            </a>
+          </h4>
+        </li>
+        <li class="section-headline">
+          <h4 class="section-headline-title">
+            <a href="#">
+              The podcasts that got us through 2020                                        
+            </a>
+          </h4>
+        </li>
+        <li class="section-headline">
+          <h4 class="section-headline-title">
+            <a href="#"> 
+              How Netflix shapes mainstream culture, explained by data                    
+            </a>
+          </h4>
+        </li>
+        <li class="section-headline">
+          <h4 class="section-headline-title">
+            <a href="#"> 
+              Christmas is better with Tom Vasel’s YouTube board game reviews                    
+            </a>
+          </h4>
+        </li>
+      </ul>
+    </section>
+  </section>
+  <!-- Latest deals section -->
+  <section>
+    <div class="container">
+      <div class="general-title">
+        <h2 class="mt-50 latest-news-main-title">
+          <a class="decoration-none" href="#">
+            Latest deals
+            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
+              <path fill="none" stroke="#F42" stroke-width="3" stroke-miterlimit="10" d="M4.75 1.5l6.5 6.5-6.5 6.5" />
+            </svg>
+          </a>
+        </h2>
+      </div>

```

Comment:
The link (`a` tag) to "See all deals" is in the same row that the title but aligned to the right.
Let's include the link on the right. 
![Screenshot from 2021-02-25 14-34-42](https://user-images.githubusercontent.com/44072711/109207238-e5316d00-7776-11eb-83bf-3561ecc39431.png)

Reply:
Okay, thanks for your review. we'll work on them.